### DISCUSSION TITLE
label style on an edge like hover in css?

### DISCUSSION THREAD

**Discussion by Yeenyeong** [bcakmakoglu/vue-flow#238] [Q&A] [answered]
I've used EdgeText to implement a label on an edge. And the static style of the label has been set. Now I want to set the style when the mouse is over the label. Something like 'hover' in css. 
Is it supported by the project?

**Comment by bcakmakoglu** [comment, marked as answer]
Can you give me an example?
You're using the `EdgeText` component in a custom edge? is that right?

You can either handle mouseover events like any other element
```vue
<EdgeText @mouseenter="onMouseEnter" @mouseleave="onMouseLeave" />
```

or use css like this for example:
```css
 .vue-flow__edge-textwrapper:hover .vue-flow__edge-text {
    fill: red;
  }

**Reply by Yeenyeong** [reply]
`<EdgeText :label="my label" :label-style={ fill: white} />`
And what I want is that when mouse is over the text on the label, the text is decorated by underline or something like that.

**Reply by bcakmakoglu** [reply]
Did you try the suggestion I gave?

**Reply by Yeenyeong** [reply]
I've changed my approach to achieve the feature. But I think your suggestion makes sense.
